config.go

```diff
@@ -35,6 +35,7 @@ const (
 	defaultRPCHost            = "localhost"
 	defaultMaxPendingChannels = 1
 	defaultNumChanConfs       = 1
+	defaultTrickleDelay       = 10
```

Comment:
`10ms` as a default trickle delay is far too aggressive. It should instead be within the range of 30 - 60 _seconds_. It was only set low before this PR as we needed snappiness within the integration tests. With this PR now, then a lower value (in the `ms`) is appropriate for integration test runs.